templates/role.tmpl

```diff
@@ -27,36 +27,128 @@
-                    Permissions: <ol>
+                    Permissions:
+                    <ol>
                         {{range .Permissions}}
                             <li style='list-style-type: none;'><span class='tab'></span>
                                 {{if $permissionAdmin}}
-                                    <a href="/internal/permission/{{.PermissionID}}">{{.Name}}</a>
+                                    <a href="/internal/permission/{{.PermissionID}}">{{.Name}}</a>&emsp;<a
+                                            class="button is-danger is-outlined"
+                                            onclick="removePermission{{.PermissionID}}FromRoleModal()">Remove
+                                        permission</a>
                                 {{else}}
-                                    {{.Name}}
+                                    {{.Name}}&emsp;<a class="button is-danger is-outlined"
+                                                      onclick="removePermission{{.PermissionID}}FromRoleModal()">Remove
+                                        permission</a>
                                 {{end}}
                             </li>
+                            <div id="removePermission{{.PermissionID}}FromRoleModal" class="modal">
+                                <div class="modal-background"></div>
+                                <div class="modal-content">
+                                    <div class="box">
+                                        <article class="media">
+                                            <div class="media-content">
+                                                <div class="content">
+                                                    <p class="title">Are you sure you want to remove "{{.Name}}" from
+                                                        this role?<br>
+                                                        Be careful! Removing permissions can have unintended
+                                                        consequences.</p>
+                                                    <form action="/internal/role/{{$.Role.RoleID}}/permission/remove/{{.PermissionID}}"
+                                                          method="post">
+                                                        <button class="button is-danger">Remove permission</button>
+                                                    </form>
+                                                </div>
+                                            </div>
+                                        </article>
+                                    </div>
+                                </div>
+                                <button class="modal-close is-large" aria-label="close"></button>
+                            </div>
+                            <script>
+                                function removePermission{{.PermissionID}}FromRoleModal() {
```

Comment:
This will create a function for every permission ID - are you sure we want to do that? Could make a `function removePermissionFromRoleModal(permID) {}`.

Reply:
Will also fix this